src/preact/base-element.js

```diff
@@ -541,6 +551,13 @@ export class PreactBaseElement extends AMP.BaseElement {
 
 // Ideally, these would be Static Class Fields. But Closure can't even.
 
+/**
+ * An experiment required for this component.
+ *
+ * @protected {string}
+ */
+PreactBaseElement['experiment'] = '';
+
```

Comment:
Is this implemented?

Reply:
Oops. Forgot to cleanup. I tried this in one of the iterations. But realized that the `isExperimentOn` requires strict constants (which is a good idea). So I dropped this idea for now.